rbi/sorbet/tprops.rbi

```diff
@@ -148,6 +148,7 @@ end
 module T::Props::Serializable
   def deserialize(hash, strict = nil); end
   def recursive_stringify_keys(obj); end
+  sig {params(strict: T.nilable(T::Boolean)).returns(T::Hash[String, T.untyped])}
```

Comment:
Why is this nilable? The source code defines it like this:

https://github.com/sorbet/sorbet/blob/bfae3ff4d9173587b27bb596c785f1ac08baf89b/gems/sorbet-runtime/lib/types/props/serializable.rb#L14-L18

Reply:
I was following that code + the existing RBI which had `strict = nil`; can I just make that `strict = true` to match?

Reply:
> can I just make that `strict = true` to match?

yes please!